docs/copied-from-beats/shared-docker.asciidoc

```diff
@@ -40,28 +40,184 @@ https://www.docker.elastic.co[www.docker.elastic.co].
 
 endif::[]
 
-[float]
+==== Run the {beatname_uc} setup
+
+Running {beatname_uc} with the setup command will create the index pattern and
+load visualizations, dashboards, and machine learning jobs.  Run this command:
+
+ifeval::[("{beatname_lc}"=="filebeat") or ("{beatname_lc}"=="metricbeat") or ("{beatname_lc}"=="heartbeat")]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+{dockerimage} \
+setup -E setup.kibana.host=kibana:5601 \
+-E output.elasticsearch.hosts=["elasticsearch:9200"] <1> <2>
+--------------------------------------------
+endif::[]
+
+ifeval::["{beatname_lc}"=="packetbeat"]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+--cap-add=NET_ADMIN \
+{dockerimage} \
+setup -E setup.kibana.host=kibana:5601 \
+-E output.elasticsearch.hosts=["elasticsearch:9200"] <1> <2>
+--------------------------------------------
+endif::[]
+
+ifeval::["{beatname_lc}"=="auditbeat"]
+["source", "sh", subs="attributes"]
+--------------------------------------------
+docker run \
+  --cap-add="AUDIT_CONTROL" \
+  --cap-add="AUDIT_READ" \
+  {dockerimage} \
+  setup -E setup.kibana.host=kibana:5601 \
+  -E output.elasticsearch.hosts=["elasticsearch:9200"] <1> <2>
+--------------------------------------------
+
+<1> Substitute your Kibana and Elasticsearch hosts and ports.
+<2> If you are using the hosted Elasticsearch Service in Elastic Cloud, replace
+the `-E output.elasticsearch.hosts` line with the Cloud ID and elastic password
+using this syntax:
+endif::[]
+
+[source,shell]
+--------------------------------------------
+-E cloud.id=<Cloud ID from Elasticsearch Service> \
+-E cloud.auth=elastic:<elastic password>
+--------------------------------------------
+
 ==== Configure {beatname_uc} on Docker
 
 The Docker image provides several methods for configuring {beatname_uc}. The
-conventional approach is to provide a configuration file via a bind mount, but 
+conventional approach is to provide a configuration file via a volume mount, but 
 it's also possible to create a custom image with your
 configuration included.
 
-[float]
-===== Bind-mounted configuration
+===== Example configuration file
 
-One way to configure {beatname_uc} on Docker is to provide +{beatname_lc}.yml+ via a bind mount.
-With +docker run+, the bind mount can be specified like this:
+Download this example configuration file as a starting point:
+
+["source","sh",subs="attributes,callouts"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/beats/deploy/docker/{beatname_lc}.docker.yml
+------------------------------------------------
+
+===== Volume-mounted configuration
```

Comment:
Need to add APM Server config

Reply:
i don't know how important is this. if it is much trouble, i'd skip it too. pinging @graphaelli again for a 2nd thought.

Reply:
Since we already need to make changes on the beats side, let's get this fixed rather than eliminating it, as it is a very common way to use our image.  The apm-server block will be identical to the one for heartbeat.  To be clear, something like:
```
ifeval::[("{beatname_lc}"=="heartbeat") or ("{beatname_lc}"-="apm-server")]
```

Reply:
Thanks! Added in 05eb70d